apps/shipments/views/note.py

```diff
@@ -61,7 +62,14 @@ def get_queryset(self):
 
     def perform_create(self, serializer):
         if settings.PROFILES_ENABLED:
-            created = serializer.save(user_id=self.request.user.id, shipment_id=self.kwargs['shipment_pk'])
+            username = get_requester_username(self.request).split('@')[0]
+            org_name = get_organization_name(self.request)
+            if org_name:
+                username = f'{username} [{org_name}]'
```

Comment:
Instead of formatting/storing this on the backend I think it probably makes more sense to store both username and org name in the note and let the frontend do the formatting. This will be more flexible for future formatting changes.

Reply:
:+1:  Sounds good!